examples/platform/nxp/common/wifi_connect/include/WifiConnect.h

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright 2023-2024 NXP
+ *  All rights reserved.
+ *
+ *  SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __WIFICONNECT_H__
+#define __WIFICONNECT_H__
+namespace chip {
+namespace NXP {
+namespace App {
+
+/*
+ * Function allowing to join a Wi-Fi network based on Wi-Fi build credentials
+ * Must be called after completing Wi-Fi driver initialization
+ */
+void WifiConnectAtboot(void);
```

Comment:
```suggestion
void WifiConnectAtboot();
```

we generally try to remove the extra `void` inside brackets since clang-tidy has helped us be consistent that way (it knows how to remove everywhere, but not how to add everywhere, so it is easier to stay consistent and not have that void)